notes/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+from uuid import uuid4
+
+# Create your models here.
+
+class Note(models.Model):
+    title = models.CharField(max_length=200)
+    content = models.TextField(blank=True)
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
```

Comment:
Try thinking of a different resource your API can deal with. I made an inventory for cards from a trading card game, another student is doing one on stock prices. This is a great opportunity to revisit and explore SQL concepts we learned near the end of the FSW portion.